care/emr/api/viewsets/scheduling/booking.py

```diff
@@ -89,6 +104,23 @@ def get_queryset(self):
             .order_by("-modified_date")
         )
 
+    @classmethod
```

Comment:
Why is it a classmethod?

Reply:
i just tried to stick with similar pattern as of create appointment. will revert 😅 

![image](https://github.com/user-attachments/assets/0f2827ab-dc68-4b2a-97d2-3b65c71bdaca)

Reply:
These are classmethods because they are called from the OTP based viewsets as well.